Readme.md

```diff
@@ -10,11 +10,29 @@ API documentation is generated at runtime and lives in `/documentation`
 
 Clone this project:
 
-`git clone https://github.com/atb-as/bff-oneclick-planner.git`
+`git clone https://github.com/atb-as/atb-bff.git`
 
-Build and run the docker image:
+### Docker
 
-`docker run --rm --env PORT=8080 -p 8080:8080 -it $(docker build -q .)`
+Build and run the docker image, for local development:
+
+```
+docker build --target=dev -t atb-bff:dev .
+docker run --rm -it -e PORT=8080 -p 8080:8080 -v $PWD:/app atb-bff:dev
+```
+
+### Starting locally
+
+#### Requirements
+- Node.js 14.x ([Hapi doesn't support newer versions](https://stackoverflow.com/a/65489848/5976426))
```

Comment:
Ikke nødvendigvis relevant for akkurat dette, men dette er vel ikke sant lengre etter at jeg har oppdatert? 🤔

Reply:
Kanskje det du? Kan teste i staging

Reply:
Ja, sjekker lokalt her nå.

Reply:
Tenker jeg merger den her nå og ikke venter dog

Reply:
Ikke noe blokkerende nei